### PR TITLE
BladeFM + BoltFM + Some cleanup of mine and other presences

### PR DESCRIPTION
**Pull request by callumok2004**
90m0j 1
0izgm 1

### CODE REVIEW

File: websites/B/BladeFM/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      "value": "Streaming to %listeners% listeners",
+      "placeholder": "%listeners%"
+    },
+    {
+      "id": "state",
+      "title": "Change Line #2 (Presenter Details)",
+      "icon": "fas fa-microphone",
+      "value": "%songText%",
+      "placeholder": "%artist% - %song%"
+    }
```
Comment (Bas950) [inline websites/B/BladeFM/dist/metadata.json:44]: list all available options in placeholder, and allow them to use all in both lines

File: websites/B/BoltFM/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      "value": "Streaming to %listeners% listeners",
+      "placeholder": "%listeners%"
+    },
+    {
+      "id": "state",
+      "title": "Change Line #2 (Presenter Details)",
+      "icon": "fas fa-microphone",
+      "value": "%songText%",
+      "placeholder": "%artist% - %song%"
+    }
```
Comment (Bas950) [inline websites/B/BoltFM/dist/metadata.json:44]: same here

**Review by Bas950 — CHANGES_REQUESTED** [review]
No need for the changes in Glitch, Google Classroom, Google Domains please undo

**Comment by callumok2004** [comment]
> No need for the changes in Glitch, Google Classroom, Google Domains please undo

cleaning code wont hurt you, its not gonna be undone, no need

File: websites/B/BladeFM/dist/metadata.json [hunk 3]
```diff
@@ -33,15 +33,15 @@
...
-      "placeholder": "%listeners%"
+      "placeholder": "%artist%, %listeners% %songText% %title%"
     },
     {
       "id": "state",
       "title": "Change Line #2 (Presenter Details)",
       "icon": "fas fa-microphone",
       "value": "%songText%",
-      "placeholder": "%artist% - %song%"
+      "placeholder": "%artist%, %listeners% %songText% %title%"
```
Comment (Bas950) [inline websites/B/BladeFM/dist/metadata.json]: maybe add a comma after each not just the first?

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by Bas950** [comment]
> > No need for the changes in Glitch, Google Classroom, Google Domains please undo
> 
> cleaning code wont hurt you, its not gonna be undone, no need

you just removed brackets thats not really cleaning code

File: websites/B/BladeFM/dist/metadata.json (hunk 3 above)
Comment (callumok2004) [reply websites/B/BladeFM/dist/metadata.json]: shh totally didnt forget that

**Comment by callumok2004** [comment]
> > > No need for the changes in Glitch, Google Classroom, Google Domains please undo
> > 
> > 
> > cleaning code wont hurt you, its not gonna be undone, no need
> 
> you just removed brackets thats not really cleaning code

every little bit helps 🤷🏼

**Comment by Bas950** [comment]
> > > > No need for the changes in Glitch, Google Classroom, Google Domains please undo
> > > 
> > > 
> > > cleaning code wont hurt you, its not gonna be undone, no need
> > 
> > 
> > you just removed brackets thats not really cleaning code
> 
> every little bit helps 🤷🏼

not needed IMO

**Comment by callumok2004** [comment]
> > > > > No need for the changes in Glitch, Google Classroom, Google Domains please undo
> > > > 
> > > > 
> > > > cleaning code wont hurt you, its not gonna be undone, no need
> > > 
> > > 
> > > you just removed brackets thats not really cleaning code
> > 
> > 
> > every little bit helps 🤷🏼
> 
> not needed IMO
not needed but also not gonna kill anyone, its nicer to have nice code, cleaner, smaller, etc.. I dont see what ur problem with it is

**Comment by ririxidev** [comment]
Literally, can you freaking stop? It isn't place to argue.

**Review by ririxidev — CHANGES_REQUESTED** [review]
Okay you should check these things,
- BladeFM:
  - I really want to check statistics for this website, listeners etc. It is required since some time (I mean it's required by me, I don't know by others.). If you have some info how many listeners it have everyday, please give them to us on dm or github if it isn't a problem.
  - I will not repeat again and again about buttons - it isn't allowed for radio presences too, if you will do this next time, I will need to block you to make any other pull request since you don't follow guidelines.
image
- BoltFM:
  - I really want to check statistics for this website, listeners etc. It is required since some time (I mean it's required by me, I don't know by others.). If you have some info how many listeners it have everyday, please give them to us on dm or github if it isn't a problem.
   - I will not repeat again and again about buttons - it isn't allowed for radio presences too, if you will do this next time, I will need to block you to make any other pull request since you don't follow guidelines.
image
- Repeat
   - I will not repeat again and again about buttons - it isn't allowed for radio presences too, if you will do this next time, I will need to block you to make any other pull request since you don't follow guidelines.
image

Thanks,
ririxi (Contact: mail@ririxi.dev, ririxi#2721)

**Comment by callumok2004** [comment]
> Okay you should check these things,
> 
> * BladeFM:
>   
>   * I really want to check statistics for this website, listeners etc. It is required since some time (I mean it's required by me, I don't know by others.). If you have some info how many listeners it have everyday, please give them to us on dm or github if it isn't a problem.
>   * I will not repeat again and again about buttons - it isn't allowed for radio presences too, if you will do this next time, I will need to block you to make any other pull request since you don't follow guidelines.
>     image
> * BoltFM:
>   
>   * I really want to check statistics for this website, listeners etc. It is required since some time (I mean it's required by me, I don't know by others.). If you have some info how many listeners it have everyday, please give them to us on dm or github if it isn't a problem.
>   * I will not repeat again and again about buttons - it isn't allowed for radio presences too, if you will do this next time, I will need to block you to make any other pull request since you don't follow guidelines.
>     image
> * Repeat
>   
>   * I will not repeat again and again about buttons - it isn't allowed for radio presences too, if you will do this next time, I will need to block you to make any other pull request since you don't follow guidelines.
>     image
> 
> Thanks,
> ririxi (Contact: [mail@ririxi.dev](mailto:mail@ririxi.dev), ririxi#2721)

righto pal, so first off, repeat is not ANYTHING to do with this pr, so you do not need to cry about it
ALSO, the buttons are NOT going to the main page, so another reason NOT to cry


FOR OBVIOUS REASONS
- You are not allowed to review any of my pull requests
- this is just to personal conflict, with you clearing hating me
- You have made this extremely clear, especially what you said on twitter.

**Comment by ririxidev** [comment]
Sorry for misreading the links for the buttons. My fault here, but with this thing me & Bas noticed another thing that it is redirecting directly to stream. It isn't good thing really to redirect directly to stream, so please remove these buttons anyway. It isn't because our conflict. I'm just doing my job. 
> You are not allowed to review any of my pull requests.
Um, I didn't get any info about that I can't review your pull requests.

**Comment by callumok2004** [comment]
@ririxidev hard to dm someone who has blocked me
and unblock me on github otherwise i cant re-request review

**Comment by callumok2004** [comment]
have the reviewers just died? fantastic looooool

**Review by Bas950 — CHANGES_REQUESTED** [review]
Please provide listener stats for BladeFM and BoltFM so make sure they meet our requirements.
Please also do the things I suggested before.

**Review by Bas950 — CHANGES_REQUESTED** [review]
Like requested before, undo the changes to Google Classroom and Google Domains since they are useless

**Comment by Bas950** [comment]
> Please provide listener stats for BladeFM and BoltFM so make sure they meet our requirements.

 @callumok2004 has provided proof that they are.

**Comment by ririxidev** [comment]
Waiting...

**Review by StrikerFRFX — CHANGES_REQUESTED** [review]
As requested by Bas [previously](https://github.com/PreMiD/Presences/pull/3268#pullrequestreview-617764925), please undo your "cleaning" changes as they are unnecessary

File: websites/G/Google Classroom/presence.ts [hunk 4]
```diff
@@ -96,4 +96,4 @@ presence.on("UpdateData", async () => {
     presenceData.details = (await strings).settings;
   }
   presence.setActivity(presenceData);
-});
```
Comment (StrikerFRFX) [inline websites/G/Google Classroom/presence.ts:99]: Fix linebreak

**Review by StrikerFRFX — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]
LGTM

**Review by StrikerFRFX — APPROVED** [review]
lgtm

**Comment by callumok2004** [comment]
Why didn't it AUTO MERGE before I made changes...... undid that push anyways.. please RE REVIEW lol @Bas950 @StrikerFRFX

**Review by ririxidev — APPROVED** [review]
Lgtm

**Review by Bas950 — APPROVED** [review]
LGTM

**Pull request merged**